src/Integration.UnitTests/LocalServices/ProjectSystemFilterTests.cs

```diff
@@ -160,56 +169,80 @@ public void ProjectSystemFilter_IsAccepted_SonarTestProjectProperty_ReturnsPrope
         }
 
         [TestMethod]
-        public void ProjectSystemFilter_IsAccepted_IsKnownTestProject_ReturnsFalse()
+        public void ProjectSystemFilter_IsAccepted_SupportedNotExlucdedProject_IsKnownTestProject()
         {
             // Setup
             var testSubject = this.CreateTestSubject();
 
             var project = new ProjectMock("knownproject.csproj");
+            project.SetBuildProperty(Constants.SonarQubeExcludeBuildPropertyKey, "false"); // Should evaluate test projects even if false
             project.SetCSProjectKind();
-            project.SetTestProject();
+
+            // Case 1: Test not test project kind, test project exclude not set
+            project.SetBuildProperty(Constants.SonarQubeTestProjectBuildPropertyKey, ""); // Should not continue with evaluation if has boolean value
 
             // Act
             var result = testSubject.IsAccepted(project);
 
             // Verify
+            Assert.IsTrue(result, "Project not a known test project");
+
+            // Case 2: Test project kind, test project exclude not set
+            project.SetTestProject();
+
+            // Act
+            result = testSubject.IsAccepted(project);
+
+            // Verify
             Assert.IsFalse(result, "Project of known test project type should NOT be accepted");
+
+            // Case 3: test project exclude == false, should take precedence over project kind condition
```

Comment:
"test project exclude" is confusing two concepts: it's mixing up _SonarQubeTestProject_ and _SonarQubeExclude_. Same in line 209.